include/tallies/legendre.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef LEGENDRE_H
+#define LEGENDRE_H
+
+#include <iostream>
```

Comment:
```suggestion
#include <cstdint>
```
We need to include this for access to `std::size_t`.